src/components/base/OakIcon/OakIcon.tsx

```diff
@@ -52,6 +52,9 @@ export const oakIconNames = [
   "warning",
   "lightbulb",
   "lightbulb-yellow",
+  "signed-video-4",
+  "video-3",
+  "quiz-6",
```

Comment:
These match the names of the assets in Figma, but i'm feeling that this is going to become a quagmire. 

Feels like we need a better reference in Figmaland to map these properly.

Reply:
Could you speak to Mike about these ?

Reply:
@carlmw the names of these svg icons have now been updated in the Figma design kit to the following:
- intro
- video
- quiz

I'll check through other svgs to make sure naming makes sense.